contracts/service-registry/src/lib.rs

```diff
@@ -2,6 +2,10 @@ pub mod contract;
 pub mod error;
 pub mod helpers;
 pub mod msg;
-pub mod state;
+mod state;
+
+pub use state::{
+    AuthorizationState, BondingState, Service, Verifier, WeightedVerifier, VERIFIER_WEIGHT,
```

Comment:
Should we consider defining public types in `types.rs` module instead of re-exporting definitions from state?

Reply:
Some of these types are being returned as part of queries as well

Reply:
I prefer re-exporting